App/src/main/java/com/gluonhq/chat/model/Channel.java

```diff
@@ -0,0 +1,25 @@
+package com.gluonhq.chat.model;
+
+import javafx.collections.ObservableList;
+
+public class Channel {
+
+    private ObservableList<User> members;
```

Comment:
Does this needs to be an ObservableList, or would a regular list be enough? I'm not sure myself. Having an ObservableList makes this a bit heavier, and less portable, but if we have a view where we immediately bind members/messages to e.g. a ListView, than an ObservableList is super handy.

Reply:
Channel shouldn't be a part of this PR. Will update.